charts/secrets-store-csi-driver-provider-aws/templates/rbac.yaml

```diff
@@ -41,4 +49,8 @@ metadata:
   namespace: {{ .Release.Namespace }}
   labels:
 {{ include "provider.labels" . | indent 4 }}
+  {{- with .Values.rbac.serviceAccount.annotations }}
+  annotations:
```

Comment:
Curious, what's your use case for needing [annotations](https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/)?

Reply:
@simonmarty sorry for the delay in response, we link the role-arn via an `rbac.serviceAccount.annotations.eks\\.amazonaws\\.com/role-arn` annotation (here's the terraform code with the forked helm chart):

```
# AWS helm chart missing required serviceaccount iam role annotation and clusterrole secret permissions
# https://github.com/aws/secrets-store-csi-driver-provider-aws/pull/162
resource "helm_release" "secrets_store_csi_driver_aws" {
  name      = "secrets-store-csi-driver-provider-aws"
  chart     = "../../../atg-xdr-helm/environments/secrets-store-csi-driver-provider-aws"
  namespace = var.namespace

  set {
    name  = "rbac.serviceAccount.annotations.eks\\.amazonaws\\.com/role-arn"
    value = module.secrets_manager_role.iam_role_arn
  }
}
```